### DISCUSSION TITLE
Supports capturing network packets as a data source

### DISCUSSION THREAD

**Discussion by bitcapybara** [vectordotdev/vector#19507] [General]
Hi, there! It seems that `Vector` does not support capturing network packets from a specific network interface as `Source`, any plans on it? This way we can analyze network traffic and store the results, like src/dest ip address/port, tcp flags... and so on, which may be useful to monitor network traffic in some situations

**Comment by jszwedko** [comment]
Hey! No current plans to add this, but it is an interesting idea.

**Comment by bitcapybara** [comment]
OK...If there is a need, I hope to contribute my efforts.😊
